### ISSUE TITLE
Implement sharing API

### ISSUE THREAD

**Issue by ThePerkinrex** [HermesMessenger/Hermes#109]
This is so you can share something on your phone, and Hermes shows up as an option.
I don't know how the API is exactly used, but I think it only works for installed web apps

**Comment by KeyKoder** [comment]
Searching a bit I found this:
https://developers.google.com/web/updates/2018/12/web-share-target

How yo register your app as a share target, holpe it is useful!

**Comment by ThePerkinrex** [comment]
Yes, that article is made by the one who gave the talk about this on the Google I/O 2019, where I saw it mentioned
